yarn-project/p2p/src/config.ts

```diff
@@ -354,6 +366,21 @@ export const p2pConfigMappings: ConfigMappingsType<P2PConfig> = {
     printDefault: () =>
       'AuthRegistry, FeeJuice.increase_public_balance, Token.increase_public_balance, FPC.prepare_fee',
   },
+  // TODO: should P2P_MAX_TX_POOL_COUNT/P2P_MAX_TX_POOL_SIZE be configurable or a global constant?
+  // TODO: what should we do if the value is 0? Should we allow unbounded mempools?
```

Comment:
I'm fine with either unbounded (if it helps testing) or just failing to initialize